### ISSUE TITLE
clean speech is suppressed completely with df3 model, while df2 model is ok

### ISSUE THREAD

**Issue by flytair** [Rikorose/DeepFilterNet#388]
hi,
thanks for your amazing work, and you are so nice to open source your work.
when i compare the performance of your released pre-trained df2 and df3 model, i find sometimes, the clean speech(high snr) is suppressed totally by df3, it seems df3 models treats it as interference, while the df2 model works normally. please refer to the plot below:
![image](https://github.com/Rikorose/DeepFilterNet/assets/30205746/e75b7b57-972d-4c8c-8cdb-e9501f75cef1)

the noisy, output of df2 and df3 model:
[badcase.zip](https://github.com/Rikorose/DeepFilterNet/files/12036139/badcase.zip)

and if add 15db gain on the noisy speech, the over suppression by df3 model is alleviated.

i wander what is the cause of the different performance, is it caused by the architecture difference between df2 and df3 or different training set, df3 training set includes speech interference as noise?

**Comment by Rikorose** [comment]
Could you describe, how you used the df3 model? I cannot reproduce this behavior. Works fine for me.

**Comment by flytair** [comment]
hi Rikorose,
thanks for your quick response.
i run the enhance.py script with command line:  ./badcase/test.wav -o ./badcase -m ./models/DeepFilterNet3
the df code version is v0.5.3, and i have attached the code, model and test.wav
[df.zip](https://github.com/Rikorose/DeepFilterNet/files/12047003/df.zip)

**Comment by flytair** [comment]
> Could you describe, how you used the df3 model? I cannot reproduce this behavior. Works fine for me.
hi Rikorose,
thanks for your quick response.
i run the enhance.py script with command line: ./badcase/test.wav -o ./badcase -m ./models/DeepFilterNet3
the df code version is v0.5.3, and i have attached the code, model and test.wav
[df.zip](https://github.com/Rikorose/DeepFilterNet/files/12047003/df.zip)

**Comment by Rikorose** [comment]
Yes this is indeed a failure case of the df3 model. I will provide a new df3 model soon which includes a minor bugfix and has a more robust speech detection for silent voices.

**Comment by flytair** [comment]
hi Rikorose,
you are so kind to update new model, and i am looking forward it. thanks!
i am training a smaller df2/df3 model with microsoft dns3 noise dataset, but both of the smaller model over suppressed the speech, but your released df2 model works fine, and regarding this i have 2 questions:
1. i checked the dns noise dataset, and found there are some speech like noise, do you think if removing these speech like noise, would benefit speech over suppression? 
2. is the noise training set of the df2 and df3 model the same?
thanks!

**Comment by Rikorose** [comment]
1. Probably yes, but it depends on the noise
2. Its the same. The new model will include two main changes: 1. trained with low amplitude input (config: add gain with -18dB). 2. Increase SNR loss, so it robustly detects the speech signal and thus does not oversuppess it.

**Comment by flytair** [comment]
hi Rikorose,
thanks for your quick reply.
what do you mean "it depends on the noise"? could you please give more detail about it? 
thanks!

**Comment by Rikorose** [comment]
babble noise contains voices and is typically classified as noise. Thus the model needs this kind of noise during training. What noise did you remove for instance?

**Comment by flytair** [comment]
hi Rikorose,
i most want to remove traffic, air condition, car engine noise, and it can tolerate the voice like noise. do you think it is ok to remove these voice babble noise, and would it degrade the noise(excluding the voice babble noise) suppression performance?
thanks!

**Comment by Rikorose** [comment]
Sure, than remove the babble noise from the training.

**Comment by flytair** [comment]
hi Rikorose,
thanks for your patience on this issue.
looking forward to you new df mode.
thanks!

**Comment by veera-puthiran-14082** [comment]
hi Rikorose, Thanks for fixing the issue. can you share what are all the action taken to fix the issue, it would be helpful. thanks in advance

**Comment by veera-puthiran-14082** [comment]
@Rikorose
